iidm-ddb/iidm-ddb-eurostag-import-export/pom.xml

```diff
@@ -81,6 +81,23 @@
       </dependency>
 
        <dependency>
+           <groupId>eu.itesla_project</groupId>
+           <artifactId>iidm-impl</artifactId>
+           <version>0.1-SNAPSHOT</version>
+           <scope>test</scope>
+       </dependency>
+       <dependency>
+           <groupId>eu.itesla_project</groupId>
+           <artifactId>iidm-test</artifactId>
+           <version>0.1-SNAPSHOT</version>
+           <scope>test</scope>
+        </dependency>
```

Comment:
The version of ipst-core dependencies should not be specified in sub modules but use the version defined in the main pom.xml